packages/wrangler/src/__tests__/helpers/write-worker-source.ts

```diff
@@ -0,0 +1,31 @@
+import * as fs from "node:fs";
```

Comment:
Moved this helper function from Publish to this new file thinking I was going to need it for new tests in `pages.test` file, however, ended up relying on existing tests in examples in monorepo. 

I can revert this, seems like something that will eventually be abstracted.

Reply:
Yes - please revert this.